lmfdb/lfunctions/Lfunctionutilities.py

```diff
@@ -740,6 +740,34 @@ def specialValueTriple(L, s, sLatex_analytic, sLatex_arithmetic):
 
     return [lfunction_value_tex_analytic, lfunction_value_tex_arithmetic, Lval]
 
+
+##################################################################
+#Function to help display Lvalues when scientific notation is used
+##################################################################
+
+def scientific_notation_helper(lval_string):
+    efind1 = lval_string.find('e')
+    if efind1 > -1:
+        efind2 = lval_string[efind1+1:].find('e')
+        if efind2 > -1:
+            efind2 = efind1+efind2+1
+            l_beg = lval_string[:efind1]
+            l_mid = lval_string[efind1+1:efind2]
+            l_end = lval_string[efind2+1:]
+            new_real = l_beg+'\\times 10^{'+l_mid[:l_mid.find(' ')]+'}'
+            new_im = l_mid[l_mid.find(' '):] + '\\times 10^{' + l_end[:l_end.find('i')]+'}i'
+            new_l_string = new_real+new_im
+        else:
+            l_beg = lval_string[:efind1]
+            l_end = lval_string[efind1+1:]
+            new_real = l_beg + '\\times 10^{' + l_end[:l_end.find(' ')]+'}'
```

Comment:
Is there always a space at the end?  What if there's something useful after the space?

I worry that this code is too sensitive to the exact format of the input.  What about something like the following?
```python
lval_string = re.sub(r"[Ee](-?\d+)", r" \times 10^{\1}", lval_string)
```

Reply:
The regular expression substitution looks good to me, although it is always
necessary to check these things on actual examples.

For the math mode delimiters, I suggest changing them to \( and \),
which is more commonly used in the LMFDB (and perhaps we should
consider only using those and adding that to the coding standards).

Reply:
Thanks for the comments. @roed314 I had not worked with Python's re library before so I built it specific to the output of display_complex; I think yours looks much cleaner and more robust; I will do some tests as David suggests and then incorporate. @davidfarmer I will switch to the \( and \) standard. 

Sorry for the bad PR.

Reply:
It's not a bad PR!  We're just trying to make it better. 😄